inventory/group_vars/OSEv3/general.yml

```diff
@@ -0,0 +1,11 @@
+---
+openshift_pkg_version: "-0.0.1"
```

Comment:
Previously it was _only_ possible to build Origin RPMs at version `0.0.1`. Now we build more correct versions. I think we should set `openshift_release=v1.5` and make sure we are installing the RPMs that we built and not some other RPMs on accident